docs/content.go

```diff
@@ -577,14 +577,14 @@ Enterprise Performance Computing (EPC)`
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	// search
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-	SearchUse   string = `search [search options...] <search query>`
+	SearchUse   string = `search [search options...] <search_query>`
 	SearchShort string = `Search a Library for images`
 	SearchLong  string = `
```

Comment:
Since collections are removed, I think we can remove that and shorten things up.

`Search a container library for users and containers matching the search query.`